File: Custodial.BoilerPlate.Core/Custodial.BoilerPlate.Core/Service Settings/DatabaseSettings.cs
using Custodial.BoilerPlate.Core.Interfaces;
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Text;

namespace Custodial.BoilerPlate.Core.Service_Settings
{
    public class DatabaseSettings : IDatabaseSettings
    {
        public string connectionString { get; set; }
        public DatabaseTypes typeOfDatabase { get; set; } = DatabaseTypes.InMemoryDatabase;
        public string database { get; set; }
        public string collection { get; set; }
    }
}

File: FrontEndv3/Webserverv2/src/Classes/CustodialAddressingServices/DatabaseSettings.ts
import {IDatabaseSettings} from '../../Interfaces/CustodialAddressingServices/IDatabaseSettings';
export class DatabaseSettings implements IDatabaseSettings{
    public connectionString: string;
    public database: string;
    public collection: string;

    constructor(connectionString : string, database: string, collection: string){
        this.connectionString = connectionString;
        this.database = database;
        this.collection = collection;
    }
}   